src/desktop/apps/artwork2/client.tsx

```diff
@@ -77,19 +77,46 @@ mediator.on("openAuctionAskSpecialistModal", options => {
 })
 
 mediator.on("openViewInRoom", options => {
-  const url = options.image && options.image.url
-  if (url) {
-    const fakeImg = $('<img src="' + options.image.url + '">').load(function() {
-      // TODO: Give it a position so that it can animate from it.
-      // Or alternatevely figure out how to remove the animation
-    })
+  try {
+    const { dimensions } = options
+    const { url, width, height } = options.image
+
+    let newWidth = width
+    let newHeight = height
+
+    const bounds = document
+      .querySelector("[data-type=artwork-image]")
```

Comment:
For the artwork with multiple images there will be several items that this selector will get. Does it still work? I guess the bounds for all of them will be the same...